src/helpers/wincher-helper.php

```diff
@@ -14,23 +13,55 @@
  */
 class Wincher_Helper {
 
+	/**
+	 * Holds the Options Page helper instance.
+	 *
+	 * @var Options_Helper
+	 */
+	protected $options;
+
+	/**
+	 * Options_Helper constructor.
+	 *
+	 * @param Options_Helper $options The options helper.
+	 */
+	public function __construct( Options_Helper $options ) {
+		$this->options = $options;
+	}
+
 	/**
 	 * Checks if the integration is active for the current user.
 	 *
-	 * @return bool Whether or not the integration is active.
+	 * @return bool Whether the integration is active.
 	 */
 	public function is_active() {
-		// If the integration is disabled, Wincher should not be active.
-		$conditional = new Wincher_Conditional();
+		return ! $this->is_disabled() && $this->options->get( 'wincher_integration_active', true );
+	}
+
+	/**
+	 * Checks if the integration should be disabled for the current user.
+	 *
+	 * @param bool $return_conditional If the conditional class name that was unmet should be returned.
+	 *
+	 * @return bool|string Whether the integration is active.
+	 */
+	public function is_disabled( $return_conditional = false ) {
+		$conditional = new Non_Multisite_Conditional();
+
 		if ( ! $conditional->is_met() ) {
-			return false;
+
+			if ( $return_conditional === true ) {
+				return ( new \ReflectionClass( $conditional ) )->getShortName();
+			}
+
+			return true;
 		}
 
 		if ( ! \current_user_can( 'publish_posts' ) && ! \current_user_can( 'publish_pages' ) ) {
-			return false;
+			return true;
 		}
 
-		return (bool) WPSEO_Options::get( 'wincher_integration_active', true );
+		return false;
```

Comment:
Thinking about this below in the `after_integration_toggle`. If you just check the conditional there, the need for this function goes away. You can leave the `is_active` as it was before (maybe the option helper still for ease of testing of course).